kratos/python/add_variable_utils_to_python.cpp

```diff
@@ -471,6 +471,21 @@ void AddVariableUtilsToPython(pybind11::module &m)
                             const Variable<double>&,
                             const Vector&,
                             const unsigned int>(&VariableUtils::SetSolutionStepValuesVector))
+        .def("GetValuesVector", py::overload_cast<
+                            const ModelPart::NodesContainerType&,
+                            const Variable<array_1d<double,3>>&,
+                            const unsigned int>(&VariableUtils::GetValuesVector))
```

Comment:
of the 4, this is the only method that is strictly different from others. 
I am adding the 4 with consistency with the methods just above